### PR TITLE
Dumb Sabaton Memes mod

### PR DESCRIPTION
**Pull request by chaosvolt**
* Adds breeches because knee-high boots are annoying and clash with pants. Fell free to yoink and put in CDDA if anyone feels like it, historical renactor could use it.
* Adds a cavalry saber, Polish-style lobster-tail helm, and big fancy wings. Amusingly doesn't clash with wing mutants.
* Carolean profession and als-AND THEN THEN THE WINGED HUSSARS ARRIVED. Need to figure out how added light-plate cuirass and arm guards, that's as close as I could get to what I could find of hussar armor.
* Copy-from abuse to add cavalry saber to fencing. Already tested it alongside Arcana to make sure it doesn't break anything. :V

Also thanks to @Dangernoodle for that copy-from idea for martial arts, funny that it still hasn't broke yet. Also wishing I could pester someone for info on whether the hussar stuff is even even halfway decent, hard to get good info on uniforms and such. Tempted to pester @Coolthulhu but dunno how much of a history buff they are.

### CODE REVIEW

**Comment by DangerNoodle** [comment]
Amusing. The split of plate armor components would be from https://github.com/CleverRaven/Cataclysm-DDA/pull/18246.

Breeches may be useful, as there are few underwear items that have a warmth of ten, and fancy ones would serve an interesting morale role. I do not recall what the Revolutionary War reenactor professions use for the leg slot, but I would assume they are ahistorical?

Regarding hussar equipment, I am not familiar with them either. I do know that the game has a lobster helmet, but it is a modern helmet with plastic and does not include a nasal.

**Comment by chaosvolt** [comment]
Ah, neat. Didn't know who did that. 3:

And yeah, I checked reenactor and they use leather pants. Leather. Pants. Imagine some guy that's half revolutionary-war soldier and half Judas Priest.

**Comment by DangerNoodle** [comment]
Elected to import the breeches and cavalry saber for the time being, in pull request: https://github.com/CleverRaven/Cataclysm-DDA/pull/21084

One observation I made in importing the changes, the recipe for cavalry saber uses the subcategory for piercing weapons, instead of cutting weapons as expected.

**Comment by chaosvolt** [comment]
Nice. And doh. That'd be the result of lazy copy-paste from the rapier recipe. :V

File: Sabaton_memes/items.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    "techniques": [ "WBLOCK_2", "RAPID" ],
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_SWORD" ]
+  },
+  {
+    "id": "cavalry_sabre_fake",
+    "type": "GENERIC",
+    "category": "weapons",
+    "name": "cavalry saber",
+    "description": "This is a dull, cheapr replica of a curved sword associated with cavalry, from the Early Modern period onwards.",
```
Comment (DangerNoodle) [inline Sabaton_memes/items.json:26]: Typograpical error observed by @Night-Pryanik here. Cheapr instead of cheap.

**Comment by DangerNoodle** [comment]
The pull request has been merged. Breeches, cavalry sabers, and fake cavalry sabers are a part of the base game. Cavalry sabers are additionally usable in fencing with no modifications required.

Breeches were changed to the normal layer at the request of Kevin. The layering penalty is minimal however, a total of plus 4. Far less than the encumbrance lost from exchanging the leather pants for breeches.

**Comment by chaosvolt** [comment]
...this sads the dragon. I'd gripe about it, but eh. 4 extra encumbrance. ._.

**Pull request merged**